### PR TITLE
Chrome e2e tests in 1.x

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: https://jira.rax.io/browse/FRMW-910

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

### CODE REVIEW

File: utils/rx-page-objects/src/rxNotify.page.js [hunk 1]
```diff
@@ -88,6 +88,10 @@ var notification = function (rootElement) {
                 return this.isDismissable().then(function (dismissable) {
                     if (dismissable) {
                         page.btnDismiss.click();
+                        // hack for chrome support -- this is for any possible
+                        // alerts that are opened by closing this notification.
+                        // also, before you check, EC.isAlertPresent does not work here.
+                        browser.sleep(500);
```
Comment (100stacks) [inline utils/rx-page-objects/src/rxNotify.page.js]: We'll want to do same check for Chrome before invoking the 500msec delay.

**Comment by 100stacks** [comment]
LGTM

**Comment by glynnis** [comment]
Design LGTM

File: utils/rx-page-objects/src/rxDatePicker.page.js [hunk 2]
```diff
@@ -330,7 +331,8 @@ var rxDatePicker = {
 
         set: function (value) {
             this.open();
-            rxSelect.initialize(this.rootElement.element(by.model('currentYear'))).select(value);
+            var slowClick = false;
+            rxSelect.initialize(this.rootElement.element(by.model('currentYear'))).select(value, slowClick);
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxDatePicker.page.js]: See https://github.com/rackerlabs/encore-ui/pull/1714#discussion_r66871329

File: utils/rx-page-objects/src/rxDatePicker.page.js [hunk 3]
```diff
@@ -310,7 +310,8 @@ var rxDatePicker = {
 
         set: function (value) {
             this.open();
-            rxSelect.initialize(this.rootElement.element(by.model('currentMonth'))).select(value);
+            var slowClick = false;
+            rxSelect.initialize(this.rootElement.element(by.model('currentMonth'))).select(value, slowClick);
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxDatePicker.page.js]: See https://github.com/rackerlabs/encore-ui/pull/1714#discussion_r66871329

File: src/elements/ActionMenu/styles/rxActionMenu.less [hunk 4]
```diff
@@ -23,7 +23,7 @@
   z-index: 99;
   position: absolute;
   top: -10px;
-  right: 15px;
+  right: 25px;
```
Comment (CITguy) [inline src/elements/ActionMenu/styles/rxActionMenu.less]: FYI: See https://github.com/rackerlabs/encore-ui/pull/1714#discussion_r66870836

**Pull request merged**